src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -1,41 +1,45 @@
 import { FC, useState } from 'react';
-import styled from 'styled-components';
+import styled from 'styled-components/macro';
 import { CSSTransition, TransitionGroup } from 'react-transition-group';
 
 import { TokenIcon } from '@components/app';
 import { useAppTranslation } from '@hooks';
 import { Text, Icon, ChevronRightIcon, Button, SearchList, SearchListItem } from '@components/common';
 import { formatUsd, humanize } from '@utils';
 
-const StyledButton = styled(Button)`
-  background: ${({ theme }) => theme.colors.txModalColors.onBackgroundVariant};
-  color: ${({ theme }) => theme.colors.txModalColors.onBackgroundVariantColor};
-  text-transform: uppercase;
+const MaxButton = styled(Button)`
+  // background: transparent;
```

Comment:
lets remove this if we are not going to use it

Reply:
Fixed in aef61c0699b977d420319fb3f99856c4aa59fcdd